ParallaxGenLib/src/ModManagerDirectory.cpp

```diff
@@ -43,6 +43,7 @@ auto ModManagerDirectory::getModFileMap() const -> const unordered_map<filesyste
 }
 
 auto ModManagerDirectory::getMod(const filesystem::path &RelPath) const -> wstring {
+    // TODO UNICODE handle unicode lower case
   auto RelPathLower = filesystem::path(boost::to_lower_copy(RelPath.wstring()));
   if (ModFileMap.contains(RelPathLower)) {
     return ModFileMap.at(RelPathLower);

```

Comment:
I think even if `to_lower_copy()` doesn't work right it doesn't matter as long as it's consistent. This is being used for key values so that's all that matters for this case